DAO.sol

```diff
@@ -715,6 +717,45 @@ contract DAO is DAOInterface, Token, TokenCreation {
         return true;
     }
 
+    function collectInheritedDAORewards(bool _toMembers) external noEther returns (bool _success) {
+        if (msg.sender != address(this))
+            throw;
+
+        address daoAddress = privateCreation;
+
+        while (daoAddress != 0) {
+            DAO parentDAO = DAO(daoAddress);
+            parentDAO.retrieveDAOReward(_toMembers);
+            daoAddress = parentDAO.privateCreation();
+        }
+
+        return true;
+    }
+
+    function transferRewardTokens(address _to, uint _amountNum, uint _amountDen) external noEther returns (bool _success) {
```

Comment:
with this you make rewardTokens freely tradeable, and DAO tokens value becomes very hard to distinguish.